src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepository.java

```diff
@@ -39,4 +39,134 @@
 	 * @return List of {@link MostFailedContent}
 	 */
 	List<MostFailedContent> mostFailedByDefectCriteria(String launchName, String criteria, int limit);
+
+	/**
+	 * Launch statistics content loading
+	 *
+	 * @param filter        Filter
+	 * @param contentFields Fields with restrictions
+	 * @param limit         Results limit
+	 * @return List of {@link Launch}
+	 */
+	List<Launch> launchStatistics(Filter filter, Map<String, List<String>> contentFields, int limit);
```

Comment:
i would suggest extracting xxxStatistics to separate LaunchStatisticsRepository class. Do you think it makes sense?

Reply:
It could be done in the future.